### DISCUSSION TITLE
How to export Data Model and relational data

### DISCUSSION THREAD

**Discussion by tech-dfltd** [DaxStudio/DaxStudio#1193] [General]
Is there an easy way I can export the data model and relationships so I can copy and paste into chatgpt?

Thanks!

**Comment by marcosqlbi** [comment]
It depends on what ChatGPT would like to get.
If you follow the hype, you should just take a picture.
If you are serious about this thing, you need a textual description of the data model - TMDL is probably the way to go, but I'm pretty sure ChatGPT has not been trained on TMDL as it is in preview now.

**Comment by dgosbell** [comment]
From DAX Studio if you export the model metrics to a .vpax file, if you rename the .vapx file to .zip inside you should find a model.bim file. This is a json file that will have the full model metadata. I don't know if ChatGPT will understand that, but that is probably the best thing we have at the moment. (if you don't see this file go into File > Options > Vertipaq Analyzer and make sure the "Include TOM" option is enabled, then export the metrics again.
